contracts/EscrowIssuer.sol

```diff
@@ -20,14 +32,15 @@ contract EscrowIssuer is ERC20 {
     ) ERC20(_name, _symbol) {
         kwenta = IKwenta(_kwenta);
         rewardEscrow = IRewardEscrow(_rewardEscrowAddr);
+        treasury = msg.sender;
```

Comment:
Cannot be msg.sender because this will end up being the deployer of the contract (which is not the multisig)

Need to take in additional constructor param